6.3.0/aspnet-core/src/BhResturant.Application/Product/Dto/PagedProductResultRequestDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Abp.Application.Services.Dto;
+
+namespace BhResturant.Products
+{
+    public class PagedProductResultRequestDto : PagedResultRequestDto
+    {
+        public string Keyword { get; set; }
```

Comment:
what is this? why we need it?

Reply:
because we inhrite from the interface, its as to be implament.